src/utils.jl

```diff
@@ -349,17 +349,20 @@ function makeunique(names)
     set = Set(names)
     length(set) == length(names) && return Symbol[Symbol(x) for x in names]
     nms = Symbol[]
+    nextsuffix = Dict{eltype(names), UInt}()
     for nm in names
-        if nm in nms
-            k = 1
+        if nm in keys(nextsuffix)
```

Comment:
```suggestion
        if haskey(nextsuffix, nm)
```
using `haskey` is more idiomatic